lib/matplotlib/backend_tools.py

```diff
@@ -1111,7 +1110,7 @@ def trigger(self, *args, **kwargs):
 
 def add_tools_to_manager(toolmanager, tools=default_tools):
     """
-    Add multiple tools to `ToolManager`
+    Add multiple tools to ``.ToolManager``
```

Comment:
```suggestion
    Add multiple tools to `.ToolManager`
```
Either literal or dot-prefixed reference.

Reply:
fixed